documentation/STRUCTURE.md

```diff
@@ -0,0 +1,195 @@
+
+### Community
+
+Community is obviously related to the Contributor Guide, but it goes a little bit further. The community is made up of contributors and users, industry practitioners, researchers, and students. This page should provide basic information about how to reach out in order to ask questions, provide feedback, report bugs or request features. This should certainly include communication channels.
+
+We could also write up the history of the project, its roadmap, and how we envision governance to work. The code of conduct should certainly be mentioned as well.
+
+Some projects have a page showing the maintainers as well:
+
+- [scikit-learn](https://scikit-learn.org/stable/about.html#people)
+- [pandas](https://pandas.pydata.org/about/team.html)
+
+## Required steps
+
+1. Get GitHub Pages page/repository up and running
+1. Set up CI to deploy current documentation there automatically
+1. Set up CI to make documentation changes viewable, i.e. the generated HTML pages need to be visible (CircleCI)
+1. Establish webpage section as outlined above (Quickstart, User guide, etc.)
+1. Convert existing content, including reformatting markdown as ReST.
+    - We already have the examples gallery thanks to Adrin's work on sphinx-gallery: https://fairlearn.readthedocs.io/en/latest/index.html There will be plenty of work to convert existing notebooks to case studies (or user guides) as mentioned in earlier sections. If this is very laborious we can consider shortcuts for the short-term such as linking to GitHub notebooks, or using a Jupyter plugin for sphinx.
+    - Related: Document notebook development process (see separate section below)
+1. Write remaining content for all of them.
+1. We need to find a way to present the dashboard in a website where it can't be interactive. Perhaps with screenshots for the user guides, but the case studies are downloadable as Jupyter notebooks. [Could we perhaps pre-calculate all metrics and show the interactive dashboard in the example notebooks? There may be a sphinx extension for typescript]
+1. integrate landing page (will be provided by a designer), everything else from the Fairlearn project repositories
+1. add style template from ?
+1. deploy through fairlearn.org
+    - check that all pages are reachable through fairlearn.org
+    - https for fairlearn.org (currently only http works)
+    - remove API doc from readthedocs
+1. ensure the navigation from homepage to the other sections works
+    - manual testing?
+    - automated testing? (also check for broken links)
+1. Add an example notebook to show how to use estimators from various packages with our mitigation techniques.
+
+The repository structure should look similar to [what scikit-learn has](https://github.com/scikit-learn/scikit-learn/tree/master/doc):
```

Comment:
hello from https://github.com/fairlearn/fairlearn-proposals/issues/2#issuecomment-624818267!  I'm going to wait for the deploy process to land so as to avoid coupling that work with the substance of articulating things like "what is the vision of the project" and "how can outside contributors collaborate on notebooks or UX work."  But there is copy in https://github.com/kevinrobinson/fairlearn/pull/1/files gathered from this and the main repo if that is helpful to draw from later.

Reply:
I actually think you don't need to wait any longer :-) I've mostly established the workflow with sphinx, directory layout, and CI preview in PRs.

A little bit of this is captured in this markdown file: https://github.com/fairlearn/fairlearn/tree/master/docs
In PRs you now see a CircleCI "check" show up which should give you a preview of what the page will look like after your PR.

So given that you could totally start on a page such as "About Fairlearn". The content from your PR already looked great, so I'm sure we can get this into the right place with relatively little modification. What do you think?

Reply:
That works, too. I hope you did get an idea of the goals? Happy to talk more anytime! I want to keep the ball rolling on the UX discussions.

Reply:
@kevinrobinson I just pushed an update to the proposal. Over the last weeks your ideas about roadmap, mission, governance, etc. kept coming up in various contexts which I take as an excellent indicator that the proposal and the currently deployed webpage are missing this. There's now an "About" section in the proposal that could contain this information. Let me know whether you have thoughts. Obviously this is the proposal only, so there will be actual content to review later, but I wanted to first get everyone's opinion on the structure.

Reply:
@romanlutz thanks!  I don't think the format matters as much as the substance :)